### DISCUSSION TITLE
Dark navbar with light theme

### DISCUSSION THREAD

**Discussion by hackntest** [mantinedev/mantine#5800] [Q&A] [answered]
Hi there,

I'm a seasoned web dev but recently switched to React + Mantine to modernize an old AngularJS web app, and let me tell you, I'm loving Mantine so much, it's a real pleasure to work with.

My question is: is there an easy way to make some components dark while the rest of the app is light? For coherence with the old app, I'd like to make the navbar always dark in light and dark modes. I can always do this manually using CSS, but I'd like to ask if there is any way I could avoid styling everything from scratch.

Thanks!

**Comment by rtivital** [comment, marked as answer]
Thanks for kind words!

To answer your question: Mantine does not support different color scheme for a part of the application. If you need this feature, the best option is to override component styles using `className` prop or Styles API.

**Reply by hackntest** [reply]
Thanks for the answer! That's what I thought, but I just asked in case I missed something in the docs. CSS it is then

Keep up the good work!

**Reply by AykutSarac** [reply]
It was possible with the past versions, that's very unfortunate.
